build_tools/ci/cpu_comparison/run.py

```diff
@@ -1224,8 +1247,9 @@ def benchmark_aie_kernel_time(
         *input_args,
         f"--device={config.device_hal}",
         f"--benchmark_repetitions={n_repeats}",
-        f"--batch_size={n_kernel_runs}",
+        f"--batch_size={max(n_kernel_runs, n_reconfigure_runs)}",
```

Comment:
I don't think it makes sense in any test to have both `n_kernel_runs` and `n_reconfigure_runs` runs set and that would result in incorrect numbers I think, so maybe we check for invalid combinations and throw an error.